bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jdt.doc.user; singleton:=true
-Bundle-Version: 3.15.1700.qualifier
+Bundle-Version: 3.15.1701.qualifier
```

Comment:
jdt.doc.user has no changes, is version bump required?

Reply:
fixed, sorry for the goof-up!